spec/std/mime_spec.cr

```diff
@@ -0,0 +1,83 @@
+require "spec"
+require "mime"
+
+describe MIME do
+  it ".fetch" do
+    MIME.fetch(".html").partition(';')[0].should eq "text/html"
+    MIME.fetch(".HTML").partition(';')[0].should eq "text/html"
+
+    expect_raises KeyError do
+      MIME.fetch(".fooobar")
+    end
+    MIME.fetch(".fooobar", "default/fooobar").should eq "default/fooobar"
+    MIME.fetch(".fooobar") { "default/fooobar" }.should eq "default/fooobar"
+  end
+
+  it ".fetch?" do
+    MIME.fetch?(".html").should eq MIME.fetch(".html")
+    MIME.fetch?(".HTML").should eq MIME.fetch(".HTML")
+
+    MIME.fetch?(".fooobar").should be_nil
+  end
+
+  describe ".register" do
+    it "registers new type" do
+      MIME.register(".Custom-Type", "text/custom-type")
+
+      MIME.fetch(".Custom-Type").should eq "text/custom-type"
+      MIME.fetch(".custom-type").should eq "text/custom-type"
+      MIME.extensions("text/custom-type").should eq [".Custom-Type"]
+
+      MIME.register(".custom-type2", "text/custom-type")
+      MIME.extensions("text/custom-type").should eq [".Custom-Type", ".custom-type2"]
+
+      MIME.register(".custom-type", "text/custom-type-lower")
+      MIME.fetch(".custom-type").should eq "text/custom-type-lower"
```

Comment:
what happens when you fetch `.Custom-Type` here?

Reply:
The same as before. I can add a spec.